hadoop-ozone/interface-client/src/main/proto/OmClientProtocol.proto

```diff
@@ -1347,6 +1362,10 @@ message ListKeysResponse {
     repeated KeyInfo keyInfo = 2;
 }
 
+message ListKeysLightResponse {
+    repeated BasicKeyInfo basicKeyInfo = 2;
```

Comment:
QQ: Is it intentionally to set `basicKeyInfo` to 2 or just copy/paste thing.

Reply:
It is due to copy-paste from `ListKeysResponse`, changed it to `1`